testkit-java/src/main/scala/com/akkaserverless/javasdk/testkit/impl/StubActionContext.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.akkaserverless.javasdk.testkit.impl;
+
+import com.akkaserverless.javasdk.action.ActionContext;
+import com.akkaserverless.javasdk.Metadata;
+import java.util.Optional;
+import com.akkaserverless.javasdk.ServiceCallFactory
+import com.akkaserverless.javasdk.valueentity.ValueEntityContext
+import com.akkaserverless.javasdk.action.ActionCreationContext
+import akka.stream.Materializer
+
+/** Context for action calls. */
+final class StubActionContext extends ActionContext {
+
+  override def metadata() = Metadata.EMPTY
```

Comment:
Could be better to throw saying it is not supported yet.

Reply:
done now